tests/Controller/EnMarche/ExplainerControllerTest.php

```diff
@@ -25,7 +23,7 @@ public function testSuccessfulActions(string $path)
         $crawler = $this->client->request(Request::METHOD_GET, $path);
 
         $this->assertResponseStatusCode(Response::HTTP_OK, $this->client->getResponse());
-        $this->assertSame(4, $crawler->filter('.explainer__articles li')->count());
+        $this->assertSame(4, $crawler->filter('.explainer__articles > ul > li')->count());
```

Comment:
without this: we were just lucky that the total count of sub li elements were matching the final count.

This is the right way to count all the elements, now all king of articles are properly loaded.